### PR TITLE
[FIXES #759] fix german translations

### LINKED ISSUE #759: fix german translations
part of #679 for de-DE translation strings

### PR DESCRIPTION
**Pull request by gannebamm**
Dear reviewers. @allyoucanmap Is this the correct repo and branch and @t-book Could you please take a quick look at my translations.

### CODE REVIEW

File: geonode_mapstore_client/static/mapstore/gn-translations/data.de-DE.json [hunk 1]
```diff
@@ -95,15 +96,15 @@
...
             "uploadDocument": "Dokument hochladen",
             "addRemoteService": "Remote Service hinzufügen",
             "exploreMaps": "Karten erkunden",
             "createMap": "Karte erstellen",
             "dashboard": "Dashboard",
             "dashboards": "Dashboards",
-            "apps": "app",
-            "createApp": "Karte app",
+            "apps": "Apps",
+            "createApp": "App Erstellen",
```
Comment (t-book) [inline geonode_mapstore_client/static/mapstore/gn-translations/data.de-DE.json:107]: lowercase erstellen?

File: geonode_mapstore_client/static/mapstore/gn-translations/data.de-DE.json [hunk 2]
```diff
@@ -118,48 +119,48 @@
...
+                "noPermissions": "Sie sind nicht berechtigt eine GeoStory hinzuzufügen"
             }
         },
         "gnviewer": {
             "edit": "Bearbeiten",
             "export": "Export",
-            "editInfo": "Bearbeitungs information",
+            "editInfo": "Informationen anzeigen",
             "editMetadata": "Metadaten bearbeiten",
             "editStyle": "Stil Bearbeiten",
```
Comment (t-book) [inline geonode_mapstore_client/static/mapstore/gn-translations/data.de-DE.json:144]: lowercase "bearbeiten"?

**Review by allyoucanmap — CHANGES_REQUESTED** [review]
Hi @gannebamm @t-book, thanks for the contribution!
A clarification on the process:
the files inside `geonode_mapstore_client/static/mapstore/gn-translations/` and `geonode_mapstore_client/static/mapstore/ms-translations/` are the results of the build process so these two directories will be deleted by next build if you commit those as is right now.

You should edit this file instead https://github.com/GeoNode/geonode-mapstore-client/blob/master/geonode_mapstore_client/client/static/mapstore/translations/data.de-DE.json for the  `geonode_mapstore_client/static/mapstore/gn-translations/data.de-DE.json` 

While for `geonode_mapstore_client/static/mapstore/ms-translations/data.de-DE.json` you have two options:
- add the translations as an override integration with same id structure to https://github.com/GeoNode/geonode-mapstore-client/blob/master/geonode_mapstore_client/client/static/mapstore/translations/data.de-DE.json
- make a PR to mapstore2 repo updating this file https://github.com/geosolutions-it/MapStore2/blob/master/web/client/translations/data.de-DE.json

I noticed that a lot of translations have been removed from the ms-translation, please don't remove them if you are opting for the PR on the MapStore2 respository because the core framework is using those translations for plugin that are not included in the geonode-mapstore-client project

**Comment by allyoucanmap** [comment]
@gannebamm The changed file now is correct but I had to restore the two other ones that will be overridden by the compile process. Let me know if I can proceed with the merge, thanks!

**Comment by gannebamm** [comment]
@allyoucanmap Shall I recreate the statics, too?
I have created a PR against the main repo here: https://github.com/geosolutions-it/MapStore2/pull/7799
Is this correct?

**Comment by gannebamm** [comment]
@allyoucanmap The source file (https://github.com/GeoNode/geonode-mapstore-client/pull/760/commits/1b0843d6b7a23bd5f68f239fbf7bff1bdf9ee4ef) is fine. The compiled ones will get recreated by you in the compile process, right? Because currently, those are missing if I understand this correctly.
For me, it is fine to merge this.

**Comment by allyoucanmap** [comment]
@gannebamm once we merge this PR the github action will copy the two translations:

- GeoNode custom translations client/static/mapstore/translations/data.de-DE.json into /static/mapstore/gn-translations/data.de-DE.json
- MapStore2 translations client/MapStore2/web/client/translations/data.de-DE.json into /static/mapstore/ms-translations/data.de-DE.json

So ms-translations and gn-translations content is always updated programmatically

The MapStore2 translations will not update until we merge the PR geosolutions-it/MapStore2#7799 and update the MapStore commit reference here accordingly (This process could take time).

So if you need additional change of the MapStore2 translations I suggest you to add them into the [client/static/mapstore/translations/data.de-DE.json](https://github.com/GeoNode/geonode-mapstore-client/blob/1b0843d6b7a23bd5f68f239fbf7bff1bdf9ee4ef/geonode_mapstore_client/client/static/mapstore/translations/data.de-DE.json)

example: an override of the print legend translation

```js
{
    "locale": "de-DE",
    "messages": {
        // add translation with the same path of mapstore core translation to change them
        "print":{
            "alternatives": {
                "legend": "Legende anzeigen"
            }
        },
        // ... all the overrrides,
        "geostory": {
            "geoNode": "GeoNode"
        },
        // ...
    }
}
```

hope this clarify the process

**Review by allyoucanmap — APPROVED** [review]

**Pull request merged**